django/forms/forms.py

```diff
@@ -192,19 +192,19 @@ def add_initial_prefix(self, field_name):
         return 'initial-%s' % self.add_prefix(field_name)
 
     def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_on_separate_row):
-        "Output HTML. Used by as_table(), as_ul(), as_p()."
+        """Output HTML. Used by as_table(), as_ul(), as_p()."""
```

Comment:
I'm not sure if this was generated by your editor but please avoid cosmetic changes in such patches.

Reply:
@charettes ok, got it. Should I revert this particular line?

Reply:
Yes, please.